### DISCUSSION TITLE
Cyfrin Updraft Website Not Loading

### DISCUSSION THREAD

**Discussion by armaansidana2003** [Cyfrin/foundry-full-course-cu#2928] [General]
Im unable to access the website as its showing internal error. Is the issue only at my end for everyone else too?

**Comment by abhinavreddyyy** [comment]
It's working now. Kindly check again

**Reply by EngrPips** [reply]
Indeed.

**Reply by divineat123** [reply]
Mine is not working

**Comment by divineat123** [comment]
I have accessed the website and tried a course but it’s not playing

**Reply by EngrPips** [reply]
Hello @divineat123, Are you still facing this issue?

**Comment by mishael1655** [comment]
Mine is very slow,furst I thought it was my network connection,, but id not.Other site are opening on my phone.It's difficult to learn when It's taking hours to load.
pls do something

**Reply by EngrPips** [reply]
Are you using `vpn`?

**Reply by mishael1655** [reply]
no

**Reply by EngrPips** [reply]
You can try another internet service provider and see if that will make any difference.

**Reply by mishael1655** [reply]
its working now ,I tried another internet service provider
Thank u

**Reply by EngrPips** [reply]
I am glad you figured it out. Keep crushing it.
